### PR TITLE
[WebProfiler] Fixed logo and menu profiler for Microsoft Edge

### PR DESCRIPTION
**Pull request by WhiteEagle88**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #16887 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Icon/symfony.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="40" height="24" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
```
Comment (fabpot) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Icon/symfony.svg]: Shouldn't it be 24 as well here?

**Comment by WhiteEagle88** [comment]
@fabpot, No. If you inspect logo in chrome, you can see 40x40. Maybe need change height?

**Comment by fabpot** [comment]
@WhiteEagle88 As the logo is square, both width and height should be the same IMHO.

**Comment by WhiteEagle88** [comment]
@fabpot 
Yes, I agree with you. I fixed this.

For symfony 3.0 will need to create pull request?

**Comment by javiereguiluz** [comment]
Could any Windows user please verify this fix? Thanks!

**Comment by javiereguiluz** [comment]
We need some Windows user to verify this fix before merging. @WouterJ? @iltar? Thanks!

**Comment by WouterJ** [comment]
I'm sorry, I don't have Windows 10 so no Edge for me.

**Comment by AlexandrePavy** [comment]
Works well for me, Windows 10 Microsoft Edge 25.10586.0.0 with EdgeHTML 13.10586

profiler

**Comment by javiereguiluz** [comment]
@AlexandrePavy thanks for confirming the fix! And thanks to @WhiteEagle88 for fixing it.

**Comment by fabpot** [comment]
Thank you @WhiteEagle88.

**Pull request merged**